src/sif/autocost.cc

```diff
@@ -8,6 +8,7 @@
 #include "midgard/constants.h"
 #include "midgard/util.h"
 #include "sif/dynamiccost.h"
+#include <assert.h>
```

Comment:
should be `#include <cassert>` always prefer the `c` prefixed header name vs the `.h` version